### ISSUE TITLE
surround, but name it autofeetplace so its chad B)

### ISSUE THREAD

**Issue by HausemasterIssue** [Sxmurai/inferno#14]
screw the feature suggestion normie format. the client dosen't have a surround and i want one in it, so maybe make it? i have some suggestions for the surround. something that i think it needs is legit switch when placing obi. silent switch desyncs u on 2b, so u need to do legit switch, good thing ik a client that does that, cookie client: https://github.com/bebeli555/CookieClient/blob/main/src/main/java/me/bebeli555/cookieclient/mods/combat/Surround.java. this surround will switch to obi legit then place and switch to ur old slot when done, u can use this code for inspiration ig. also make sure once the surround is turned on it will turn off the ca (which is kinda ass atm), place the obi, then turn ca back on agian if it was on. also momentum has some strict surround features like ncp strict rotations which might come in handy. so ye, add surround except make it actually good on strict.

**Comment by Sxmurai** [comment]
can you use the format when making an issue in the future, its there for a reason. its so i can actually read it and quickly figure out what you want easier

**Comment by Sxmurai** [comment]
and im actually doing autocrystal right now, so surround is coming after it

**Comment by HausemasterIssue** [comment]
ok

the only real points i wanted to hit on were:

making the surround switch to obsidian, then place, then switch back, legitimately, not silent.

if the ca is on and surround is on, ca will turn off, then back on after surround is finished.

**Comment by Sxmurai** [comment]
fair enough, why should it be called AutoFeetPlace > Surround, which would you rather have:

1. Surround
2. AutoFeetPlace
3. Do a Future and name it AutoObsidian

which of the three would you rather have? im thinking classic surround

**Comment by HausemasterIssue** [comment]
hmmm idk tbh, i dont like surround that much cuz every client names it that, but def either AutoFeetPlace or AutoObsidian, or maybe even AutoSurround? either one works, its not that big of a deal.

**Comment by Sxmurai** [comment]
done in commit https://github.com/Sxmurai/inferno/commit/bd71f3ecb283b6b3619cedcfdcda005823fec5a4
